pip._vendor.urllib3.exceptions.InvalidChunkLength.__repr__
937c4b2c06727d3ed20259ccc976c250